README.md

```diff
@@ -216,7 +216,7 @@ function readFile(filename, callback) {
 if (DEBUG) { require('debug'); }                           /*error Unexpected require().*/
 
 // a require() in a switch statement is also flagged
-switch(x) { case '1': require('1'); break; }  
+switch(x) { case '1': require('1'); break; }
```

Comment:
These changes to the README are due to the addition of .editorconfig, which happen because of `trim_trailing_whitespace`